content/sensu-go/6.3/observability-pipeline/observe-entities/entities.md

```diff
@@ -328,14 +401,36 @@ In contrast to annotations, you can use labels to filter [API responses][14], [s
 Limit labels to metadata you need to use for response filtering and searches.
 For complex, non-identifying metadata that you will *not* need to use in response filtering and searches, use [annotations][20] rather than labels.
 
+### Agent entity labels {#agent-entities-managed}
```

Comment:
This is not new -- I just moved it up so that we list the sections alphabetically by class (agent, proxy, service).